main/WebServer.cpp

```diff
@@ -8129,15 +8140,16 @@ namespace http {
 						bAllowDeviceToBeHidden = true;
 					}
 
-					if (order == "")
+					if (order == "" || (!isAlpha))
```

Comment:
I know some other have been peeking/poking in this function as well, but i prefer checking against .empty()